### PR TITLE
Manta v4201 hotfix release

### PR DESCRIPTION
**Pull request by ghzlatarev**
## Description

* Update manta weights: https://github.com/Manta-Network/Manta/actions/runs/5375565436
* Bump spec versions

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [ ] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: runtime/manta/src/weights/frame_system.rs [hunk 1]
```diff
@@ -59,51 +59,51 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
...
-			.saturating_add(Weight::from_ref_time(2_050).saturating_mul(b.into()))
+			.saturating_add(Weight::from_ref_time(2_035).saturating_mul(b.into()))
 	}
 	// Storage: System Digest (r:1 w:1)
 	// Storage: unknown [0x3a686561707061676573] (r:0 w:1)
 	fn set_heap_pages() -> Weight {
-		// Minimum execution time: 7_882 nanoseconds.
-		Weight::from_ref_time(8_752_000)
+		// Minimum execution time: 18_083 nanoseconds.
+		Weight::from_ref_time(18_888_000)
```
Comment (Dengjianping) [inline runtime/manta/src/weights/frame_system.rs:78]: I'm fine with the 2x bigger weight number.

File: runtime/manta/src/weights/pallet_parachain_staking.rs [hunk 2]
```diff
@@ -84,51 +84,51 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_parachain_staking::WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking InflationConfig (r:1 w:1)
 	fn set_staking_expectations() -> Weight {
-		// Minimum execution time: 18_810 nanoseconds.
-		Weight::from_ref_time(19_350_000)
+		// Minimum execution time: 26_699 nanoseconds.
+		Weight::from_ref_time(40_902_000)
```
Comment (Dengjianping) [inline runtime/manta/src/weights/pallet_parachain_staking.rs:88]: Fine with 2x as well,it's not a normal call.

**Review by Dengjianping — DISMISSED** [review]

**Review by ferrell-code — COMMENTED** [review]
might want to redo manta weights as the force calls were benchmarked locally on my computer

**Comment by ghzlatarev** [comment]
> might want to redo manta weights as the force calls were benchmarked locally on my computer

Ran the CI, you can reference it in the description.

**Review by ferrell-code — DISMISSED** [review]
Ah yeah don't know how I didn't see that

**Comment by ghzlatarev** [comment]
> Ah yeah don't know how I didn't see that

Well, actually maybe i can change it in the pallet as well. Will make the changelog have 4.2.0-1 manually.

**Comment by ghzlatarev** [comment]
actually @ferrell-code can you finish this PR off and change the SBT reserve price to something smaller than 100k MANTA

**Review by ferrell-code — APPROVED** [review]
Might need to update changelog-tool to handle this version style.

**Review by Dengjianping — APPROVED** [review]

**Pull request merged**